bucket/grepwin.json

```diff
@@ -0,0 +1,81 @@
+{
+    "version": "1.9.1",
+    "description": "Regular expression search and replace tool for Windows.",
+    "homepage": "https://tools.stefankueng.com/grepWin.html",
+    "license": "GPL-3.0-only",
+    "notes": "Run 'grepwin-install-context' to add grepWin to right-click context menu. \n(This does not require admin rights)",
+    "architecture": {
+        "32bit": {
+            "url": [
+                "https://github.com/stefankueng/grepWin/releases/download/1.9.1/grepWin-1.9.1_portable.exe#/grepWin_portable.exe",
+                "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/grepwin-install-context.reg",
+                "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/grepwin-uninstall-context.reg"
+            ],
+            "hash": [
+                "015a9427d29be42dfa0defc73aa613a44d6eb3b0f5b76152f988d941fb8d9868",
+                "a73beb571c5f98b2e2adea463f2823f4b01fe277eab68745c1823d9c39397b6d",
+                "4e99b868f8078fad8faee220c214dbb3288848109ae515fd50eca205b44252c5"
+            ]
+        },
+        "64bit": {
+            "url": [
+                "https://github.com/stefankueng/grepWin/releases/download/1.9.1/grepWin-x64-1.9.1_portable.exe#/grepWin_portable.exe",
+                "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/grepwin-install-context.reg",
+                "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/grepwin-uninstall-context.reg"
+            ],
+            "hash": [
+                "95335b4ad1e42149a8e5e3ce4ce7114ba8cc50d5d671209a917c4aa6b8eca653",
+                "a73beb571c5f98b2e2adea463f2823f4b01fe277eab68745c1823d9c39397b6d",
+                "4e99b868f8078fad8faee220c214dbb3288848109ae515fd50eca205b44252c5"
+            ]
+        }
+    },
+    "installer": {
+        "script": [
+           "$app_path = \"$dir\\grepWin_portable.exe\".Replace('\\', '\\\\')",
+           "$reg_content = Get-Content \"$dir\\grepwin-install-context.reg\"",
+           "$reg_content = $reg_content.Replace('$app_path', $app_path)",
+           "$reg_content | Set-Content -Path \"$dir\\grepwin-install-context.reg\"",
+           "$script_content = @(\"",
```

Comment:
You do not need to save it into ps1 scripts. 

These files are easily clickable from terminal

Reply:
I'll remove the PS1 scripts, and prompt users to run **reg import $dir\grepwin-install-context.reg** (as it doesn't require admin rights)